### DISCUSSION TITLE
유니티 기준 class 구분 기준

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#15] [Q&A] [answered]
저번에 설명드린 json파싱을 만들던 도중에 유니티에서는 스크립트하나를 컴포넌트처럼 관리하니까 전부 클래스를 구분하는게 좋은지 클래스 마다 전부 cs를 하나씩 만드는게 좋은지 헷갈립니다.  

파싱데이터가 향수, 손님, 대화 이런 클래스가 있다고 할 때 전부 .cs로 구분해서 만들어야 하는지 아니면 하나로 매핑해서 해당 cs안에 class를 구분해서 만드는게 좋은지..

**Comment by jongfeel** [comment, marked as answer]
하나의 class에
하나의 cs 파일로 만드는 걸 추천합니다.

MonoBehaviour를 상속 받은 class는 어쩔 수 없지만
그렇지 않은 class도 일관되게 관리하면 좋긴 합니다.
enum, interface 들도 마찬가지입니다.
